ios/RNAccengageModule/RNAccengageModule.m

```diff
@@ -25,17 +25,17 @@ @implementation RNAccengageModule
 #pragma mark - Permissions
 
 RCT_EXPORT_METHOD(
-                  hasPermissions:(RCTResponseSenderBlock)callback
```

Comment:
This is a callback, not a promise:
https://facebook.github.io/react-native/docs/native-modules-ios.html#callbacks
https://facebook.github.io/react-native/docs/native-modules-ios.html#promises